Dockerfile

```diff
@@ -1,24 +1,35 @@
-FROM python:3.6.6-alpine3.6
+FROM linuxserver/beets:nightly
 LABEL maintainer="Senan Kelly <senan@senan.xyz>"
-COPY \
-    ./requirements-docker.txt \
-    ./_docker_entry \
-    /
-RUN \
-    apk add --no-cache libev build-base libffi-dev sudo && \
-    pip --no-cache-dir install -U \
-        betanin \
-        https://github.com/caronc/apprise/archive/master.zip \
-        -r /requirements-docker.txt && \
-    apk del build-base
-VOLUME [ \
-    "/root/.local/share/betanin/" \
-    "/root/.config/betanin/" \
-    "/root/.config/beets/" \
-]
-ENV \
-    PYTHONUNBUFFERED=1 \
-    PYTHONWARNINGS="ignore:Unverified HTTPS request" \
-    UID=0 \
-    GID=0
-CMD [ "/_docker_entry" ]
+
+COPY root/ /
+
+RUN apk add --no-cache --virtual=build-deps --upgrade \
+        libev \
+        build-base \
+        libffi-dev \
+        git \
+        python3-dev
+
+# Remove problematic dependency
+RUN pip3 uninstall -y enum34
+
+# Install Betanin
+RUN pip3 install -U \
+    betanin \
+    https://github.com/caronc/apprise/archive/master.zip \
+    https://github.com/beetbox/beets/archive/master.zip
+
+# Disable Beets web UI
+RUN rm -rf /etc/services.d/beets
+
+# Clean up
+RUN apk del --purge \
+        build-deps
+RUN rm -rf /tmp/*
+
+VOLUME \
+    "/config/.config/beets" \
+    "/config/.config/betanin" \
+    "/config/.local/share/betanin" \
```

Comment:
unfortunately I think changing the betanin/beets data paths would be a breaking change for users of the container
eg. I personally have in my compose file
```
volumes:
- ./data:/root/.local/share/betanin/
- ./config:/root/.config/betanin/
- ./beets:/root/.config/beets/
```

though I do agree `/config` is a better place for this stuff.
maybe some symlinks could help for some backwards compatibility?